CHANGELOG.rst

```diff
@@ -70,6 +70,9 @@ time or change existing behaviors in order to make them less surprising/more use
   namespace in which doctests run.
   Thanks `@milliams`_ for the complete PR (`#1428`_).
 
+* New ``--doctest-report`` option available to change the output format of diffs
```

Comment:
Yes, sounds weird but we like to have people acknowledged in the CHANGELOG. A maintainer could do it before/after merging, but then we couldn't just use GitHub's merge button so we ask people to prepare the final CHANGELOG message so we can speed things up. :grin: 

About the missing link, you just have to add the proper link reference... scroll down in the CHANGELOG and you will see a long list of links. Just add your link there and linting should now work.